packages/contracts-bedrock/src/dispute/OutputBisectionGame.sol

```diff
@@ -284,34 +291,34 @@ contract OutputBisectionGame is IOutputBisectionGame, Clone, ISemver {
         // INVARIANT: Local data can only be added if the game is currently in progress.
         if (status != GameStatus.IN_PROGRESS) revert GameNotInProgress();
 
-        // TODO(clabby): Include less context. See Adrian's proposal for the local context salt.
-        (ClaimData storage starting, ClaimData storage disputed) = findStartingAndDisputedOutputs(_execLeafIdx);
-        bytes32 uuid = keccak256(abi.encode(starting.claim, starting.parentIndex, disputed.claim, disputed.parentIndex));
+        (Claim starting, Position startingPos, Claim disputed, Position disputedPos) =
+            findStartingAndDisputedOutputs(_execLeafIdx);
+        Hash uuid = computeLocalContext(starting, startingPos, disputed, disputedPos);
```

Comment:
Shouldn't we just be using `findLocalContext` here?
```suggestion
        Hash uuid = findLocalContext(_execLeafIdx);
```

Reply:
We unfortunately need access to the `starting` claim + position as well as the disputed claim below, so unfortunately not. Would be nice if we could, though

Reply:
MSTM